LogMonitor/src/LogMonitor/ProcessMonitor.cpp

```diff
@@ -374,3 +422,16 @@ DWORD ReadFromPipe(LPVOID Param)
 
     return ERROR_SUCCESS;
 }
+
+std::wstring ProcessMonitor::ProcessFieldsMapping(_In_ std::wstring fileFields, _In_ void* pLogEntryData)
+{
+    std::wostringstream oss;
+    ProcessLogEntry* pLogEntry = (ProcessLogEntry*)pLogEntryData;
+
+    if (Utility::CompareWStrings(fileFields, L"TimeStamp")) oss << pLogEntry->currentTime;
+    if (Utility::CompareWStrings(fileFields, L"Source")) oss << pLogEntry->source;
+    if (Utility::CompareWStrings(fileFields, L"logLine") 

```

Comment:
nit: as for the name casing, you wanted to go with PascalCase or camelCase?

Reply:
@profnandaa I have updated the function names to PascalCase and variable names to camelCase for consistency with the existing code style.